books/kestrel/apt/isodata.lisp

```diff
@@ -1101,9 +1102,45 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(define isodata-process-undefined (undefined
+                                   (old$ symbolp)
+                                   ctx
+                                   state)
+  :returns (mv erp
+               (undefined "A @(tsee pseudo-termp) that is
```

Comment:
I would use `undefined$` here, just for consistency with other code, and given that it differs from `undefined` in general.